noxfile.py

```diff
@@ -96,13 +96,22 @@ def stringify_dict(d: Dict[str, str]) -> str:
     return ", ".join(f"{k}={v}" for k, v in d.items())
 
 
-def configure_env_for_dev_build(session: nox.session) -> None:
-    """Set environment variables for a development build.
+def configure_env_for_dev_build(session: nox.Session) -> None:
+    """Set environment variables for a development test.
```

Comment:
```suggestion
    """Set environment variables for a development build.
```
(or did you explicitly want the same docstring twice?)